tests/js/index.js

```diff
@@ -73,6 +74,22 @@ exports.registerTests = function(tests) {
     }
 };
 
+exports.prepare = function(done) {
+    if (!global.isNodeProcess || global.testAdminUserInfo) {
+        done();
+    }
+
+    let helper = require('./admin-user-helper');
+    helper.createAdminUser().then(userInfo => {
+        global.testAdminUserInfo = userInfo;
```

Comment:
I guess the idea is to use only the global admin user. which is always fresh for every test run. I think cocoa reuses this user throughout their tests